packages/story-editor/src/components/library/panes/media/common/mediaElement.js

```diff
@@ -179,9 +175,15 @@ function Element({
         url={resource.attribution.author.url}
       />
     );
-
   const ref = useRef();
-
+  const onMenuCancelled = useCallback(() => {
+    if (firstElementId) {
+      document.querySelector(`[data-id='${firstElementId}']`).tabIndex = 0;
```

Comment:
Hmm doesn't feel very clean to use a data attribute to get the first element 🤔 

Is there no other way, e.g. by going via `ref.current.parentElement` or something?

Reply:
@swissspidy I think I can get the first child of the parent and set it’s tabIndex to 0 need to try. Would that look clean like either ref.current.parentElement.firstChild or either get parentRef and modify its firstChild